Integrations/GitHub/GitHub.py

```diff
@@ -494,197 +523,84 @@ def format_comment_outputs(comment: dict, issue_number: Union[int, str]) -> dict
+''' COMMANDS '''
 
-    Returns:
-        dict: API response
 
-    Raises:
-        Exception: An exception will be raised if one or more of the requested reviewers is not
-            a collaborator of the repo and therefore the API call returns a 'Status: 422 Unprocessable Entity'
-    """
-    suffix = PULLS_SUFFIX + f'/{pull_number}/requested_reviewers'
-    response = http_request('POST', url_suffix=suffix, data={'reviewers': reviewers})
-    return response
+def test_module():
+    http_request(method='GET', url_suffix=ISSUE_SUFFIX, params={'state': 'all'})
+    demisto.results("ok")
 
 
-def create_comment(issue_number: Union[int, str], msg: str) -> dict:
-    suffix = ISSUE_SUFFIX + f'/{issue_number}/comments'
-    response = http_request('POST', url_suffix=suffix, data={'body': msg})
+def create_pull_request(create_vals: dict = {}) -> dict:
+    suffix = PULLS_SUFFIX
+    response = http_request('POST', url_suffix=suffix, data=create_vals)
     return response
 
 
-def list_issue_comments(issue_number: Union[int, str]) -> list:
-    suffix = ISSUE_SUFFIX + f'/{issue_number}/comments'
-    response = http_request('GET', url_suffix=suffix)
-    return response
-
+def create_pull_request_command():
+    args = demisto.args()
+    create_vals = {key: val for key, val in args.items()}
+    maintainer_can_modify = args.get('maintainer_can_modify')
+    if maintainer_can_modify:
+        create_vals['maintainer_can_modify'] = maintainer_can_modify == 'true'
+    draft = args.get('draft')
+    if draft:
+        create_vals['draft'] = draft == 'true'
+    response = create_pull_request(create_vals)
 
-def list_pr_files(pull_number: Union[int, str]) -> list:
-    suffix = PULLS_SUFFIX + f'/{pull_number}/files'
-    response = http_request('GET', url_suffix=suffix)
-    return response
+    ec_object = format_pr_outputs(response)
+    ec = {
+        'GitHub.PR(val.Number === obj.Number)': ec_object
+    }
+    human_readable = tableToMarkdown(f'Created Pull Request #{response.get("number")}', ec_object, removeNull=True)
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
 
 
-def list_pr_reviews(pull_number: Union[int, str]) -> list:
-    suffix = PULLS_SUFFIX + f'/{pull_number}/reviews'
+def is_pr_merged(pull_number: Union[int, str]):
+    suffix = PULLS_SUFFIX + f'/{pull_number}/merge'
     response = http_request('GET', url_suffix=suffix)
     return response
 
 
-def get_commit(commit_sha: str) -> dict:
-    suffix = USER_SUFFIX + f'/git/commits/{commit_sha}'
-    response = http_request('GET', url_suffix=suffix)
-    return response
-
+def is_pr_merged_command():
+    args = demisto.args()
+    pull_number = args.get('pull_number')
 
-def add_label(issue_number: Union[int, str], labels: list):
-    suffix = ISSUE_SUFFIX + f'/{issue_number}/labels'
-    response = http_request('POST', url_suffix=suffix, data={'labels': labels})
-    return response
+    # raises 404 not found error if the pr was not merged
```

Comment:
how are we handling that then ?

Reply:
I didn't. To properly handle it would mean changing the behavior of `http_request` to raise an error instead of return_error... which should _probably_ be fine since it would be caught by the outer `except` block. but my thought at the time was that I didn't want to break functionality accidentally somewhere else in the integration code...

Reply:
👍